sites/default/sqlconf.php

```diff
@@ -1,26 +0,0 @@
-<?php
```

Comment:
You have deleted the file.  actually, just remove your changes to this file (drop from commit) and add a .gitignore file in your sites directory.  Mua, please fill him in on how *you* do this in case there is another method you prefer.
@Gicheha basically this means that for good quality, you don't actually *run* the copy of your code that is in your repo unless you are very very careful.  Write the code in your repo copy, then git pull (or manually copy/paste) your code into your test server.  This makes sure that things like user files and project files and all that don't get pulled into your PR's.  It also makes a lot of other things easier when you want to test and keep branches in new directories under your www or htdocs server directory.